core/ecschema-editing/src/Editing/Editor.ts

```diff
@@ -61,9 +66,41 @@ export class SchemaContextEditor {
   public get schemaContext(): SchemaContext { return this._schemaContext; }
 
   public async finish(): Promise<SchemaContext> {
```

Comment:
Returning the SchemaContext here, doesn't give us a lot of benefits. Since we know at that point what changes has been made to the schema, it would be probably more interesting to return the applied changes to the context in the current session. Especially as we can get the SchemaContext associated with the editor through the[ Editor's schemaContext property](https://github.com/iTwin/itwinjs-core/pull/6997/files#diff-fee38c49a6a2c6681281283ef37b7cccbc3a54af372ada73375407be15586b32R66).

Reply:
Done.